### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by ManfredKarrer**
Should be merged at public announcement on May, 16th.

### CODE REVIEW

File: _posts/2019-05-16-let-the-dao-work.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Let the DAO work"
```
Comment (m52go) [inline _posts/2019-05-16-let-the-dao-work.md]: Maybe make title something like "Time for the DAO to take over"?

File: _posts/2019-05-16-let-the-dao-work.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+author: Manfred Karrer
+excerpt: "With the DAO in place Bisq has to evolve it's organisational structure without its founders. The departure of the founders was planned from the very beginning. Now it is time to fulfill that plan.<br><br>"
```
Comment (m52go) [inline _posts/2019-05-16-let-the-dao-work.md]: it's --> its

**Review by m52go — APPROVED** [review]
utACK

File: _posts/2019-05-16-let-the-dao-work.md (hunk 1 above)
Comment (m52go) [reply _posts/2019-05-16-let-the-dao-work.md]: As it is now the title seems to imply the DAO isn't working, in a functional way.

File: _posts/2019-05-16-let-the-dao-work.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+Dear Bisq community,
+
+With the successful launch of the DAO, Bisq is entering a new phase. The bootstrapping phase where founders are steering the project forward is over and the Bisq DAO is now the infrastructure which enables that Bisq is managed by its stakeholders and contributors. The DAO was envisioned from the very beginning of the project. It was also clear from the beginning that the founders need to step back to ensure that the project can develop to its full potential as a decentralised organisation. We see in all crypto-currency projects where the founder is still around that those projects suffers from some sort of centralisation. Satoshi did Bitcoin a great favour by stepping out. Bitcoin would not be the same if he would be still around as an authority, and Bitcoin developers would not be able to develop their full potential. Of course such events might be a bit disruptive and challenging for some but it really should be seen as that what it is: **a necessary step to enable Bisq to be a decentralized autonomous organization.**
```
Comment (arunasurya) [inline _posts/2019-05-16-let-the-dao-work.md]: Needs a comma before "...and the Bisq DAO is now the infrastructure...".

File: _posts/2019-05-16-let-the-dao-work.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+So this moment might be a good occasion to look back to what we have achieved and look forward what is on our roadmap.
+
+- Bisq is the only really decentralised Bitcoin-Fiat exchange
+- We have developed the first Bitcoin based DAO
+- We built up a strong community of contributors
+- We earned a great reputation and Bisq became a quite well recognised brand in the Bitcoin community
+- Already more than 50% of Bisq traders are using BSQ already after just a few weeks. So the DAO is not only working on the contributors' side but also accepted by our traders.
```
Comment (arunasurya) [inline _posts/2019-05-16-let-the-dao-work.md]: Remove "already" after "using BSQ" (repeated twice).

Comment (arunasurya) [inline _posts/2019-05-16-let-the-dao-work.md]: Add "is" before "also accepted by our traders."

Comment (ManfredKarrer) [reply _posts/2019-05-16-let-the-dao-work.md]: Thanks. What a sloppy writer I am ;-)

**Pull request merged**